web-components/src/components/header/components/HeaderMenuItem/HeaderMenuItem.Hover.tsx

```diff
@@ -67,43 +67,41 @@ const MenuHover = ({
         )}
         {renderTitle && renderTitle()}
       </span>
-      {popoverAnchor.current !== null && (
```

Comment:
I remember doing that to prevent the dropdown to appears in the bottom right screen corner but apparently it created a new issue. I could no longer reproduce the original issue on the deploy preview so I can guess it's safe to remove now.

Reply:
oh actually just realized it does appear back again at the bottom right, but then disappears very quickly, which makes it hard to catch! so I need to look into this a bit closer...

Reply:
so looking more closely at some mui example: https://mui.com/material-ui/react-popover/#mouse-over-interaction and  also react docs regarding documentation for using refs (in particular do not read while rendering, see "Pitfall" in https://react.dev/reference/react/useRef#usage), I switched to using state but still need to keep the ref to the menu item title so the popover is still displayed below it once we hover on the dropdown elements